build.sh

```diff
@@ -32,13 +32,13 @@ while getopts qe c; do
   \?) usage;;
   esac
 done
-shift `expr $OPTIND - 1`
+shift $( expr $OPTIND - 1 )
```

Comment:
Newby question: what's the difference?

Reply:
They are same, but `$( )` has the advance that you can nest. According to http://mywiki.wooledge.org/BashFAQ/082, `$( )` is slightly preferred.